src/PrestaShopBundle/Entity/Shop.php

```diff
@@ -299,23 +297,4 @@ public function hasMainUrl(): bool
 
         return false;
     }
-
-    /**
-     * Checks if a configuration value is overridden for this shop
-     *
-     * @param ShopConfigurationInterface $configuration
-     * @param string $configurationKey
-     *
-     * @return bool
-     */
-    public function isConfigurationKeyOverridden(ShopConfigurationInterface $configuration, string $configurationKey): bool
```

Comment:
Removed method but it's not a BC as it was introduced one week ago https://github.com/PrestaShop/PrestaShop/pull/23894/